kong/plugins/session/strategies/postgres/session_metadatas.lua

```diff
@@ -0,0 +1,14 @@
+local fmt = string.format
+
+local session_metadatas = {}
+
+function session_metadatas:select_by_audience_and_subject(audience, subject)
```

Comment:
We can check if the `audience` and `subject` are `nil`. If `nil`, we scan skip the database query!

Reply:
Good idea.